pkg/local-resource-manager/controller_manager.go

```diff
@@ -38,6 +40,108 @@ import (
 // resources calculation.
 
 // Start starts the controller.
+
+func canMergeFlavours(flavour1, flavour2 *nodecorev1alpha1.Flavour) bool {
```

Comment:
Is this method meant for all kind of Flavors? I believe that it should be just used for specific Flavor types like VMs or K8s clusters.

Reply:
So I would add a check on the Flavor type at the beginning of the method to address this concern.

Reply:
We need to define the different types of Aggregation to address all the possible use cases. 

Spec.Type is already checked in the method.

Reply:
Well, I didn't mean that.

As we're going to provide SaaS as flavours, those cannot be aggregated like hardware resources.

So, when I talk about "check on the flavor Type at the beginning", I mean we should allow the aggregation only to specific resource types (VMs, K8s-clusters). Not only when flavor1 and flavo2 have the same type.

Reply:
I think that we need to add a field which refers to the specific resource type and set it in the Flavour definition.